pyminio/main.py

```diff
@@ -328,15 +470,33 @@ def _get_metadata(cls, file_pointer: File, metadata: Dict):
         return file_metadata
 
     @_validate_directory
-    def get_last_object(self, path: str):
-        bucket, directory_path, _ = self._get_match(path)
+    def get_last_object(self, path: str) -> File:
+        """Return the last modified object.
+
+        Args:
+            path: path of a directory.
+        """
+        match = Match(path)
         objects_names_in_dir = self.listdir(path, only_files=True)
         if len(objects_names_in_dir) == 0:
             return None
 
         last_object_name = objects_names_in_dir[0]
-        relative_path = self._get_relative_path(directory_path,
-                                                last_object_name)
-        new_path = join(ROOT, bucket, relative_path)
+        relative_path = join(match.prefix, last_object_name)
+        new_path = join(ROOT, match.bucket, relative_path)
 
         return self.get(new_path)
+
+
+if __name__ == "__main__":
+    minio_obj = Minio(
+        endpoint=environ.get('MINIO_CONNECTION'),
+        access_key=environ.get('MINIO_ACCESS_KEY'),
+        secret_key=environ.get('MINIO_SECRET_KEY'),
+        secure=False
+    )
+
+    pyminio = Pyminio(minio_obj=minio_obj)
```

Comment:
Conside adding class method which create Pyminio instance from credentials so the user dont need to actually define the Minio instance before

Reply:
nice suggestion

Reply:
d